src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -5,14 +5,113 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import codeu.model.data.User;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.UUID;
+import java.util.List;
+import codeu.model.store.basic.UserStore;
+import org.jsoup.Jsoup;
+import org.jsoup.safety.Whitelist;
+import codeu.model.store.basic.ConversationStore;
+import codeu.model.store.basic.MessageStore;
+import codeu.model.store.basic.UserStore;
+import codeu.model.data.User;
+import codeu.model.data.Message;
 
-
+import org.mindrot.jbcrypt.BCrypt;
 /** Servlet class responsible for the Profile page. */
 public class ProfileServlet extends HttpServlet {
+	
+	private UserStore userStore;
+	
+	private ConversationStore conversationStore;
+	
+	private MessageStore messageStore;
+	
+	@Override
+	  public void init() throws ServletException {
+	    super.init();
+	    setUserStore(UserStore.getInstance());
+	    setConversationStore(ConversationStore.getInstance());
+	    setMessageStore(MessageStore.getInstance());
+	  }
+
+	  /**
+	   * Sets the UserStore used by this servlet. This function provides a common setup method for use
+	   * by the test framework or the servlet's init() function.
+	   */
+	  void setUserStore(UserStore userStore) {
+	    this.userStore = userStore;
+	  }
+	  
+	  void setConversationStore(ConversationStore conversationStore) {
+		  this.conversationStore = conversationStore;
```

Comment:
watch out for indentation throughout this file https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation